{"uuid": "1a378214-6969-4120-9169-cc5adc4800e0", "children": ["a9b7799d-8991-43f5-8124-24def02e1e91"], "befores": [{"name": "_browser", "status": "broken", "statusDetails": {"message": "KeyError: 'capabilities'\n", "trace": "  File \"/Users/artlv/work/acld/venv/lib/python3.7/site-packages/pluggy/callers.py\", line 187, in _multicall\n    res = hook_impl.function(*args)\n  File \"/Users/artlv/work/acld/venv/lib/python3.7/site-packages/_pytest/fixtures.py\", line 958, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"/Users/artlv/work/acld/venv/lib/python3.7/site-packages/_pytest/fixtures.py\", line 784, in call_fixture_func\n    res = next(it)\n  File \"/Users/artlv/work/acld/tests/conftest.py\", line 17, in _browser\n    b = MyExtendedBrowser()\n  File \"/Users/artlv/work/acld/tests/conftest.py\", line 12, in __init__\n    super().__init__()\n  File \"/Users/artlv/work/acld/venv/lib/python3.7/site-packages/wasd/wd/browser.py\", line 27, in __init__\n    self.new_driver()\n  File \"/Users/artlv/work/acld/venv/lib/python3.7/site-packages/wasd/wd/browser.py\", line 42, in new_driver\n    desired_capabilities=SettingsManager.get('capabilities')\n  File \"/Users/artlv/work/acld/venv/lib/python3.7/site-packages/wasd/core/settings_manager.py\", line 47, in get\n    return cls._config[k]\n"}, "start": 1587731356313, "stop": 1587731356313}], "start": 1587731356313, "stop": 1587731356429}